conda/recipes/rapids_cuda_patched_dependencies/build.sh

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+# Script assumes the script is executed from the root of the repo directory
+
+REPODIR=$(cd $(dirname $0); pwd)
```

Comment:
For consistency with other names that use an `_`: 
```suggestion
REPO_DIR=$(cd $(dirname $0); pwd)
```